### ISSUE TITLE
Linux distributions

### ISSUE THREAD

**Issue by madalinignisca** [dcoxall/dcoxall-wordpress#5]
Hi, you should mention that this is only for Debian based distributions, php5-fpm is called php-fpm in rhel/rpm based distributions and the ini path is in another place (the default that php comes).

**Comment by dcoxall** [comment]
Yes. Although support for other distributions and OSs is perhaps the main thing I would love pull requests for. I am only familiar with debian based distros.
